jennchenn/uwexplore
# Added prereq/antireq info area

## Implementation description

- Added prereq/antireq section to source cards
- Added message in table if section information is not available
- Added card to show when users first visit the site or there are 0 search results 



## Steps to test

1. Verify prereq/antireq headings appear on cards (will always be none for now) 

2. Verify message in table section appears
![image](https://user-images.githubusercontent.com/52092038/224849898-f4c63926-1cbf-4f67-ae6a-db7fb18a2c4c.png)

3. Verify card appears on page load and for 0 search results
![image](https://user-images.githubusercontent.com/52092038/224849931-aac1af8a-08e0-4fee-8e98-0fea3b1aa6ac.png)




## What should reviewers focus on?

-
